.travis_dependencies.sh

```diff
@@ -34,7 +34,7 @@ if [ ! -d "$src" ]; then
 
         conda install -c conda-forge ffmpeg
 
-        pip install pysoundfile
+        pip install soundfile
```

Comment:
This is fine, but really we need to migrate this project from travis to gh actions.  As is, I don't think the tests work anymore.

Reply:
I added a simple action that installs dependencies and runs `python -m pytest tests` on `python 3.6` ~ `python 3.12`.

You can see the [result of the build](https://github.com/jhj0517/pyrubberband/actions/runs/11032672136) from the test branch.

+) This [jhj0517:fix/compatibility](https://github.com/jhj0517/pyrubberband/tree/fix/compatibility) branch is allowed to be directly edited by maintainers, so you may be able to remove old `.travis` files if you want.